lilypad/cli/commands/start.py

```diff
@@ -51,7 +51,7 @@ def start_lilypad(project_dir: Path, port: str) -> subprocess.Popen:
     env["LILYPAD_PROJECT_DIR"] = str(project_dir)
     files_dir = files("lilypad").joinpath("server")
     process = subprocess.Popen(
-        ["fastapi", "dev", "--port", port],
+        ["fastapi", "dev", "--port", str(port)],
```

Comment:
there's another issue with `lilypad.configure` where it hardcodes port 8000 and instead needs to read from the config file, please fix that

Reply:
```
    config = load_config()
    port = config.get("port", 8000)
    otlp_exporter = JSONSpanExporter(
        base_url=f"http://127.0.0.1:{port}/api",
    )
```
this is the code inside configure, are you saying its not pulling the port properly?

Reply:
oh hmm, for some reason it wasn't tracing, not sure why

Reply:
there was a bug in start that didnt set the config file properly, which may have caused a weird state.